### ISSUE TITLE
Warning of missing "use_https" while it is there.

### ISSUE THREAD

**Issue by GreenEric** [nagdevAmruthnath/minio.s3#3]
Hi, Nagdev,
Sorry, it's me again.
I'd encountered such situation for times.
It keeps telling `"use_https" is missing` while it is enrolled.
Does it mean I miss some other parameters?
Many thanks for your great support and advice.
Best regards,
Eric

```
> minio.s3::s3HTTP(query = "c1.jpg", bucket = "images", use_https = F)
Error in s3HTTP(verb = "GET", bucket = bucket, query = list(location = ""),  : 
  argument "use_https" is missing, with no default
```

**Comment by johnbrookedev** [comment]
I have the same issue, any solutions?

**Comment by pilare** [comment]
We encounter that error while using `s3read_using` function without providing the `opts` parameter. However when adding list of parameters to `opts`, it went pretty smooth. 
```
s3read_using(FUN = download.file, object = "c1.jpg", bucket = 'images', opts = list(use_https = F))
```

**Comment by johnbrookedev** [comment]
found a workaround but am too fried to figure out why. There is a bug in "minio.s3:::setup_s3_url()" where it apparently cannot find the variable "use_https". Commenting out the the following check in the function resolves the issue. It is odd as there is a default for the variable in the function call. 

```
#    if (isTRUE(use_https)) {
#        prefix <- "https://"
#    }
#    else {
        prefix <- "http://"
#    }
```

Sorry am not more help with tracking it down. Too fried atm.
package: minio.s3_0.3.8.tar.gz
platform: x86_64-w64-mingw32
R version: R version 4.0.2


Edit: well should read, thanks for the response, will check it out tomorrow. Best wishes

**Comment by GreenEric** [comment]
> found a workaround but am too fried to figure out why. There is a bug in "minio.s3:::setup_s3_url()" where it apparently cannot find the variable "use_https". Commenting out the the following check in the function resolves the issue. It is odd as there is a default for the variable in the function call.
> 
> ```
> #    if (isTRUE(use_https)) {
> #        prefix <- "https://"
> #    }
> #    else {
>         prefix <- "http://"
> #    }
> ```
> 
> Sorry am not more help with tracking it down. Too fried atm.
> package: minio.s3_0.3.8.tar.gz
> platform: x86_64-w64-mingw32
> R version: R version 4.0.2
> 
> Edit: well should read, thanks for the response, will check it out tomorrow. Best wishes

Yes, I read the source, also feel confused, "it is here, isn't it?"

**Comment by nagdevAmruthnath** [comment]
Hello all for not really being active here. When I wrote this package, I had a Minio in a local server. Hence, It was connecting to Https instead of Http. So, when using this package please mention if you are using Http or Https.

**Comment by pilare** [comment]
@GreenEric @nagdevAmruthnath we had similar problems, we end up solving this on a @vstathias fork:
https://github.com/vstathias/minio.s3
Maybe @vstathias can make a PR and merge this to @nagdevAmruthnath repo

**Comment by nagdevAmruthnath** [comment]
That would be great! I will merge it when I get PR.
